src/style-spec/package.json

```diff
@@ -9,14 +9,22 @@
     "mapbox-gl-js"
   ],
   "license": "ISC",
-  "main": "./dist/index.js",
-  "module": "./dist/index.es.js",
+  "main": "./dist/index.es.js",
```

Comment:
I don't understand the interaction of this with `"type": "module"`. Does this field even matter, or does the `"."` field in `"exports"` override this with environment-specific values?

Reply:
This affects browserify usage. I've reverted this to its original state and confirmed that browserify, node+cjs and node+esm all work with this setup. Continuing on and checking more setups…